src/libraries/System.Data.Odbc/src/CompatibilitySuppressions.xml

```diff
@@ -147,84 +147,203 @@
   <Suppression>
     <DiagnosticId>CP0001</DiagnosticId>
     <Target>T:System.Data.Odbc.ODBC32</Target>
-    <Left>runtimes/freebsd/lib/netcoreapp2.0/System.Data.Odbc.dll</Left>
+    <Left>lib/netcoreapp3.1/System.Data.Odbc.dll</Left>
```

Comment:
All these comparisons of netcoreapp3.1 vs netstandard2.0 are because we dropped the 3.1 assets from the packages in main but the previous version still has them. I will open an issue to add a feature to PackageValidation where we can specify an itemgroup with target frameworks to ignore.